### PR TITLE
change the redis resource format

### PR DESCRIPTION
**Pull request by galdor**
We remove the '*' mark for duplicate commands in a pipeline.

Before:
BenchmarkTestRedisQuantizer-4  1000000  2228 ns/op  1012 B/op  8 allocs/op

After:
BenchmarkTestRedisQuantizer-4  1000000  2045 ns/op   996 B/op  7 allocs/op

### CODE REVIEW

File: quantizer/redis_test.go [hunk 1]
```diff
@@ -73,10 +73,10 @@ func TestRedisQuantizer(t *testing.T) {
 			"GET ..."},
 
 		{"GET k1\nDE...\nGET k2",
```
Comment (LotharSee) [inline quantizer/redis_test.go]: Out of curiosity, are these tests making sense?
Is it possible to receive weirdly-truncated resources like these? `GET k...\nDEL k2\nMS...`, `GET k1\nDE...\nGET k2`, ...
I'd prefer tests to focus on real type of data, instead of testing impossible edge cases.

Comment (galdor) [inline quantizer/redis_test.go]: Both lines and pipelines can get truncated by the sender.
